src/modules/Dropdown/Dropdown.jsx

```diff
@@ -509,7 +509,7 @@ export default {
           this.item && 'item',
           this.floating && 'floating',
           this.fluid && 'fluid',
-          Object.prototype.hasOwnProperty.call(this.$options.propsData, 'pointing') && `pointing ${this.pointing}`,
+          typeof this.$options.propsData === 'object' && this.$options.propsData !== null && Object.prototype.hasOwnProperty.call(this.$options.propsData, 'pointing') && `pointing ${this.pointing}`,
```

Comment:
I still don't get this. Can't you write `this.pointing && `pointing ${this.pointing}`? If `pointing` isn't defined, it will pass `false`. The function `classes` will handle that